src/jon_mikkel_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Jon-Mikkel Korsvik"
+__email__ = "jonkors@nmbu.no"
+
+
+def bubble_sort(list_for_sorting):
+    """ Inputs a list for sorting, and transforms it into a new "sorted_list".
```

Comment:
Since you asked. The proper format here is to have a blank line between the first sentence of the docstring and then de-indent the rest.